workout-index.js

```diff
@@ -15,24 +15,47 @@ document.addEventListener('DOMContentLoaded', async function(){
     const data = await response.json();
     console.log(data);
     for (let i=0; i<data.length; i++){
-  let exerciseName = data[i].name;
-  let exerciseEquipment = data[i].equipment;
-  let exerciseMuscle = data[i].muscle;
-  let exerciseInstructions = data[i].instructions;
-  exercise.push([exerciseName, exerciseEquipment, exerciseMuscle, exerciseInstructions])
+      let exerciseName = data[i].name.replaceAll("-", " ");
+      let exerciseEquipment = data[i].equipment.replaceAll("_", "");
+      let exerciseMuscle = data[i].muscle;
+      let exerciseInstructions = data[i].instructions;
+      exercise.push([exerciseName, exerciseEquipment, exerciseMuscle, exerciseInstructions])
     }
     const grid = new gridjs.Grid({
-    columns:['Name', 'Equipment', 'Muscle', 'Instructions'],
-    data: exercise
-  }).render(document.getElementById('grid'));
+      columns:[{
+        name:'Name',
+        width: 150,
+        resizable: true,
+        sort: true,
+      }, {
+        name: 'Equipment', 
+        width: 200,
+        resizable: true,
+      }, {
+        name:'Muscle',
+        width: 150,
+        resizable: true,
+      }, {
+        name:'Instructions'
+        
+      }],
+      search: true,
+      fixedHeader: true,
+      height: '500px',
+      data: exercise,
+
+    }).render(document.getElementById('grid'));
+
+    document.getElementById("grid").style.textTransform = "capitalize";
```

Comment:
I think that it's best if you directly Capitalize the text in the lines above (18-19) instead of doing it using CSS